docs/Daos.md

```diff
@@ -0,0 +1,252 @@
+# Working with DAOs
+
+Arc.js provides a class called [DAO](api/classes/DAO) that facilitates creating and working with DAOs.  Read on for more information about DAOs in DAOstack and the [DAO](api/classes/DAO) class.
+
+## About DAOs
+
+Every DAO in DAOstack has an architecture designed by Arc that enables it to use the reusable contracts provided by Arc.  Each DAO has an avatar, a controller, a token, a reputation system, and a set of schemes and global constraints.
+
+**The avatar** is a public-facing part of the DAO that handles the interaction of the DAO with the rest of the world, for example interacting with other DAOs or paying third party agents.  A DAO is always referenced by the address of its avatar.
+
+**The controller** is a central internal hub for all of the components of the DAO.
+
+**The token** is used as currency in operations like staking in `GenesisProtocol`, vesting agreements in `VestingScheme` and rewards in `ContributionReward`.
+
+**The reputation system** provides a kind of currency of influence. Reputation conveys influence in the DAO commenserate with the rules 
+of the DAO, such as when voting and the nature of what causes it to be gained or lost.  Like tokens, reputation can be minted, but it cannot be transferred between agents.
+
+**Schemes** are public-facing contracts that any agent can use when they want to work with proposals respecting the DAO.  Accordingly, each scheme is configured with a voting machine (or like [GenesisProtocol](api/classes/GenesisProtocolWrapper), is itself a scheme).
+
+**Global constraints** use configured criteria to potentially block any action attempted by the Controller.  An example is [TokenCapGC](api/classes/TokenCapGCWrapper) that limits the total supply of a given token to a certain maximum number.
+
+DAOs are extensible beyond the reusability of all Arc contracts: You can provide your own implementation of the controller, token, and any global constraints, voting machines and schemes you want.
+
+!!! info
+    Refer here for [more information about the architecture of DAOs](https://daostack.github.io/arc/README/#the-structure-of-a-dao).
+
+!!! info
+    Refer here for [more information about proposals](Proposals).
+    
+## Creating a new DAO
+
+When creating a DAO you can configure its name, token, founders and schemes.  For schemes you configure their parameters, permissions and their voting machine.
+
+You will call [DAO.new](api/classes/DAO#new) to create a new DAO, passing it a [configuration object](api/interfaces/NewDaoConfig).
```

Comment:
> api/interfaces/NewDaoConfig 

Does this link exist ?

Reply:
Yes: https://daostack.github.io/arc.js/api/interfaces/NewDaoConfig/

Reply:
Though it is not exist on the repo itself.only on GitHub.io.
pressing on this link via GitHub.com will lead to dead end.